nca/FWRules/ConnectivityGraph.py

```diff
@@ -78,10 +78,11 @@ def _get_peer_details(self, peer, format_requirement=False):
                     peer_name = peer_name.replace(ch, to_replace[ch])
         else:
             peer_name = str(peer)
-        return peer_name, peer_type, nc_name
+        text = [t for t in peer_name.split('/') if t != nc_name][0]
```

Comment:
I didn't understand this line. Please add a comment what you do here.

Reply:
done, also change the code